sdks/python/pom.xml

```diff
@@ -200,7 +200,7 @@
                   <executable>${python.user.base}/bin/tox</executable>
                   <arguments>
                     <argument>-e</argument>
-                    <argument>ALL</argument>
+                    <argument>py27,py27gcp,py27cython,lint,docs</argument>
```

Comment:
Why this change?

Reply:
Is this so precommit excludes coverage?

Reply:
yes, coverage tool will run full set of `py27gcp` tests again which I think is unnecessary to include in Precommit considering current build time (1+hour).

Reply:
Is there any way to phrase this as a negative, so we get any possible new configurations by default? (If not, drop a comment that this is everything but coverage...)

Reply:
Found a work around way: `tox envlist` will run everything defined in [tox:envlist] so we can exclude `cover` from it. As long as new defined configuration is added to [tox:envlist], maven build will include it.